plugins/python-build/share/python-build/pypy2.7-7.3.8

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+declare VERSION='7.3.8'
+declare PYVER='2.7'
+# https://www.pypy.org/checksums.html
+# https://downloads.python.org/pypy
+
+# associative array for urls
+declare basesrc="pypy${PYVER}-${VERSION}-src"
+declare basever="pypy${PYVER}-v${VERSION}"
+declare baseurl="https://downloads.python.org/pypy/${basever}"
+declare -A urlmap=(
```

Comment:
I made the array so I could get the url and checksum from a unit test script WITHOUT installing - as installing requires matching "architecture".  I'll have to check 3.2 to see how this might be possible.